### ISSUE TITLE
Muss man sich zur O-Woche anmelden ?

### ISSUE THREAD

**Issue by woche1** [digitalmediabremen/hej#297]

**Comment by simonslang** [comment]
Die O-Woche besteht aus vielen kleinen Veranstaltungen in welchen ihr in die Struktur des Studiums eingeführt werdet und euch gegenseitig kennen lernt. Für die O-Woche generell müsst ihr euch also natürlich nicht anmelden! 

Da durch COVID-19 allerdings nur begrenz viele Personen in die HfK / Uni zugelassen werden sind einige Präsenz-Veranstaltungen wie z.B. die Erstsemesterbegrüßung am 26.10 in kleinere Gruppen aufgeteilt. Wann euer Zeitslot sein wird und ob ihr euch dafür irgendwo eintragen müsst erfahrt ihr aber rechtzeitig.

**Comment by avemew** [comment]
No! Taking part in the Orientation Week is voluntary. If you feel confident in starting your studies without a short introduction provided by us you are ofcourse free to spend your time different. If you are totaly new to the HFK or Uni Bremen we do recommend you to show up though :D
